construdelas/Thais_Bighetti/Campeoes_lol/gerando_json_lol.py

```diff
@@ -0,0 +1,67 @@
+import json
+import requests
+
+"""No meu desafio de hoje, eu queria baixar algum JSON, pegar alguns dados e salvar em um novo JSON. Encontrei o JSON 
+do jogo League Of Legends e separei as seguintes informações: Nome do campeão, Título, Posição no jogo, Fala, Dificuldade, 
+Vida, Armadura, Resistência Mágica e Velocidade de Movimento. """
+
+r = requests.get('http://ddragon.leagueoflegends.com/cdn/11.22.1/data/en_US/champion.json')
+lol = r.json()
+data = lol['data']
```

Comment:
[suggestion] já que a variável `lol` só está sendo utilizada na linha 10 o que acha de atribuir a variável data direto?

```suggestion
data = r.json()['data']
```